book/scheduling-and-threading.md

```diff
@@ -723,15 +692,141 @@ def show_count():
     return out
 ```
 
+Load this up and observe an animation from 0 to 100.
+
+However, there is a pretty big flaw in our implementation that might be apparent
+when you try this demo. Depending on your computer's speed, it might run really
+fast or really slow. In fact, the animation will take a total amount of time
+about equal to the time it takes to run 100 rendering tasks. This is
+bad---animations should have a smooth and consistent frame rate, or *cadence*,
+regardless of the computer hardware.
```

Comment:
This was missing from the chapter before and is a key addition!

Reply:
Cool